### DISCUSSION TITLE
adding a deviance to ctb scores

### DISCUSSION THREAD

**Discussion by awesometurkey832** [ppy/osu#18976] [Ideas]
hello, i would like to add an idea for osu!lazer. currently osu!std has a deviance (i think thats what that is) when you click on your score and i think thats very cool. i would like to suggest one for ctb. since ctb only has 3 scoring judgments it can just be a chart that shows when you missed at what time of the song, if you missed any droplets (can be smaller than the miss to signify it was a droplet) and when + how many bananas you collected at the song. its a small thing but i think it will be easier to identify when you are failing the most just in around 10 seconds. please let me know if i need to explain something a bit better

**Comment by peppy** [comment]
We do hope to show more statistics like this in the future.

With the existing score storage in lazer we aren't storing timestamped misses, but they are already in the replay header data (which I think we'll be storing in the future) so it's not a huge reach to see this happening.

**Comment by smoogipoo** [comment]
It's a bit of a parallel request, but we should already be able to use the existing hit events to show some cool statistics on the results screen, such as positional deviation (on the plate) of caught fruits. Those interested may look into how this property is used in osu!: https://github.com/ppy/osu/blob/master/osu.Game.Rulesets.Osu/Judgements/OsuHitCircleJudgementResult.cs#L20-L23
